teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
> I'd assume a Status / Type enumeration API. And then matching (probably exactly) by title, yes.

I see your point. I'll check this today. I just was sure these constants are global, now I'm not that sure :)